apps/chat-e2e/src/assertions/publishFolderEntityAssertion.ts

```diff
@@ -0,0 +1,47 @@
+import { FolderAssertion } from '@/src/assertions/folderAssertion';
+import { PublishingExpectedMessages, TreeEntity } from '@/src/testData';
+import { PublishFolder } from '@/src/ui/webElements/entityTree';
+
+export class PublishFolderEntityAssertion<
```

Comment:
Is that possible for us to have folders with version not in the publishing flow only? Probably we should move these asertions to the folderAssertion class?

Reply:
not, version is defined only for publishing that's why I created separated assertion class